docs/user-guide/api-mediation/api-gateway-configuration.md

```diff
@@ -158,90 +51,83 @@ To change this default configuration, include the following parameters:
+    API Mediation Gateway uses external API to map certificate to it's owner in SAF. This property let's Gateway know, where does this api reside. In Zowe, ZSS is providing this api. Provide ZSS URL in the following format: `-Dapiml.security.x509.externalMapperUrl=http://localhost:<ZSS-PORT>/certificate/x509/map`. Default port is `8542` hostname is `localhost` as ZSS server is accessible only locally.
 
-**Note:** For more information about Zuul configuration parameters, see the [Spring Cloud Netflix documentation](https://cloud.spring.io/spring-cloud-netflix/multi/multi__router_and_filter_zuul.html).
+* **apiml.security.x509.externalMapperUser**
 
-## Hystrix configuration
+    To authenticate to the mapping API, a JWT token is sent with the request. The token is representing the user, that is configured with this property. The user here should be authorized to use `IRR.RUSERMAP` resource within `FACILITY` class. Default value is `ZWESVUSR` and the permissions are setup during installation with the `ZWESECUR` jcl or workflow. If you decide to customize ZWESECUR jcl or workflow (`// SET ZOWEUSER=ZWESVUSR * userid for Zowe started task`), change the `apiml.security.x509.externalMapperUser` to new value.
 
-The API Gateway contains a Hystrix configuration as shown in the following example:
+* **apiml.security.zosmf.applid**
 
-```yaml
-hystrix:
-    command:
-        default:
-            fallback:
-                enabled: false
-            circuitBreaker:
-                enabled: false
-            execution:
-                timeout:
-                    enabled: false
-                isolation:
-                    thread:
-                        timeoutInMilliseconds: ${apiml.gateway.timeoutMillis}
-                    strategy: SEMAPHORE
-                    semaphore:
-                        maxConcurrentRequests: 100000
-```
+    When z/OSMF is used as an authentication provider, to allow client certificate authentication it is necessary to provide
+valid APPLID. The API ML authenticates to the z/OSMF via generation of the passticket and using it to authenticate to the
+z/OSMF. The value in the default installation of the z/OSMF is `IZUDFLT`. To change the value to another one use the following procedure.
 
-Hystrix is a latency and fault tolerance library designed to isolate points of access to remote systems, 
-services and 3rd party libraries, stop cascading failure, and enable resilience in complex distributed systems where failure is inevitable.
+3. Restart Zowe&trade.
 
-**Note:** For more information about Hystrix configuration parameters, see the [Netflix - Hystrix documentation](https://github.com/Netflix/Hystrix/wiki/Configuration#execution.isolation.strategy).
+## Gateway timeouts
 
-## Additional parameters
+* **apiml.gateway.timeoutMillis**
 
-* **ibm.serversocket.recover**
+    This property is used to define global value for http/ws client timeout. For finer control, you can override the properties below.
```

Comment:
Define 'finer control'?

Reply:
instead of global timeout, a timeout can be set for each individual component

Reply:
1 timeout value for everything, or many timeout values for different things

Reply:
There has been updates. Please check the new doc